### DISCUSSION TITLE
Possibility of exporting lists to CSV or similar

### DISCUSSION THREAD

**Discussion by julioasotodv** [community/community#9340] [Other Feature Feedback, Questions, & Ideas]
As much as this new lists feature is great, it becomes very hard to combine "to-read" lists for both repos and other resources (books, papers, blogs etc) that may live outside GitHub.

The ability to export a list to a simple format (such as CSV, JSON, YAML or other _almost-human-readable_ format) could make it easier to combine GitHub lists with other resources.

Thank you!

**Comment by brianlovin** [comment]
Thanks for the note @julioasotodv — agreed it would be awesome to consume list contents in other ways, maybe even with a public  API. I'll flag this for our team to think about 😊
